gen2-bounding-box-filtering/kalman/kalman_filter.py

```diff
@@ -0,0 +1,218 @@
+import depthai as dai
+import blobconverter
+import cv2
+import time
+import numpy as np
+import pandas as pd

```

Comment:
if we use formula above (isntead of parsing csv) we can also remove pandas dependency